python/mlcroissant/recipes/tfds_croissant_builder.ipynb

```diff
@@ -0,0 +1,1540 @@
+{
+  "cells": [
+    {
+      "cell_type": "markdown",
+      "metadata": {
+        "id": "n92WWWKk6o1I"
+      },
+      "source": [
+        "# TensorFlow Datasets for Croissant 🥐\n",
```

Comment:
`Train a model with Croissant and TFDS`?

Reply:
Done.